src/SAT/Impl.hs

```diff
@@ -40,12 +40,12 @@ import qualified SAT.Data (satInitialize, satStore, satSolve, satClose, satPush,
 import Tools.Config (Config)
 
 -- <><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><>
--- Uncomment to select the implementation:
+-- Uncomment to select the implementation (or you would, if there were in
+-- fact multiple working implementations; non-working code for Picosat and
+-- SBV was removed):
 
--- import SAT.Picosat -- Experimental implementation (Not working)
 import SAT.SMTLib2
--- import SAT.SBV -- This implementation is outdated. The file will be deleted
```

Comment:
Now that we are (mentally) committing to SMTLib2, we may want to simplify our design for interacting with the SAT solver.